cypress/e2e/testtab_withOneReport.cy.js

```diff
@@ -1,115 +1,107 @@
-import chaiColors from "chai-colors";
+import chaiColors from 'chai-colors';
 
 chai.use(chaiColors);
-
-describe("Tests with one report", function () {
-  beforeEach(function () {
+//Hard coded waits have to be replaced with intercepts and waits
```

Comment:
Create an issue for this:

Hard coded waits have to be replaced with intercepts and waits in `cypress/e2e/testtab_withOneReport.cy.js`

Reply:
And remove comment ofc

Reply:
Comments have been removed